README.md

```diff
@@ -61,13 +62,60 @@ catch (RiotSharpException ex)
 {
   // Handle the exception however you want.
 }
-  
+
 foreach (var stat in varusRanked.Stats)
 {
   Console.WriteLine(stat.Name + "  " + stat.Value);
 }
 ```
 
+###Tournament API
+
+You first have to apply for a tournament API key [here](https://developer.riotgames.com/).
+
+Entry point for the tournament API:
+```c#
+var tournamentApi = TournamentRiotApi.GetInstance("TOURNAMENT_API_KEY");
+```
+
+Next up, create a provider.
+The url will receive callbacks with match results.
+```c#
+var provider = tournamentApi.CreateProvider(region, url);
+```
+
+And create a tournament:
+```c#
+var tournament = tournamentApi.CreateTournament(provider.Id, "TOURNAMENT_NAME");
+```
+
+Now you can create tournament codes (which you can use to join games):
+```c#
+var tournamentCode = tournamentApi.CreateTournamentCode(tournament.Id,
+    teamSize, allowedSummonerIds, TournamentSpectatorType.All,
+    TournamentPickType.TournamentDraft, TournamentMapType.SummonnersRift,
+    string.Empty);
+```
+The tournament code can now be entered in the client to join a game with the specified settings.
+Please note that you should save the provider and tournament IDs if you wish to create more tournament codes later on. You should not create a separate provider and tournament ID for every tournament code. Make sure to read Riot's [guidelines](https://developer.riotgames.com/docs/tournaments-api) on this topic.
+
+To create a Tournament object for an existing tournament, use the following syntax:
+
+```c#
+var tournament = new Tournament { Id = id };
+```
+
+You can then create codes in two ways:
+
+```c#
+tournament.CreateTournamentCode(teamSize, allowedSummonerIds,
+	TournamentSpectatorType.All, TournamentPickType.TournamentDraft,
```

Comment:
Works fine here and on the GitHub markdown viewer, but I've changed it to 4 spaces instead of a tab, maybe that's what you meant?